src/TestCase.php

```diff
@@ -164,6 +165,10 @@ protected function tearDown()
 
             Mockery::close();
         }
+        
+        if (class_exists(Carbon::class)) {
+            Carbon::setTestNow();   
```

Comment:
remove the if condition, `Illuminate\Support\Carbon` is always available.

Reply:
I just copied it from Laravel's testcase https://github.com/laravel/framework/blob/5.5/src/Illuminate/Foundation/Testing/TestCase.php#L165 

But you prefer without the if condition?

Reply:
Give me a scenario where you can install `Illuminate\Foundation\Testing\TestCase` without installing `illuminate/support`. That condition is pointless.

Reply:
I agree :) Will remove it :)